ncm-network/src/main/perl/network.pm

```diff
@@ -303,6 +303,31 @@ sub Configure {
         ### set the networktype
         if ( $net{$iface}{'type'} ) {
             $text .= "TYPE=".$net{$iface}{'type'}."\n";
+            ### Set OVS related variables
+            if ($net{$iface}{'type'} =~ /^OVS/) {
+                $text .= "DEVICETYPE='ovs'\n";
+                if ($net{$iface}{'ovs_bridge'}) {
+                    $text .= "OVS_BRIDGE='$net{$iface}{ovs_bridge}'\n";
+                }
+                if ($net{$iface}{'ovs_opts'}) {
+                    $text .= "OVS_OPTIONS='$net{$iface}{ovs_opts}'\n";
+                }
+                if ($net{$iface}{'ovs_extra'}) {
+                    $text .= "OVS_EXTRA='$net{$iface}{ovs_extra}'\n";
+                }
+                if ($net{$iface}{'bond_ifaces'}) {
+                    $text .= "BOND_IFACES=".join(' ',$net{$iface}{bond_ifaces})."\n";
```

Comment:
@Pansanel i'm doing a review for the whole component, but this needs single quotes right?
(pls don't push a fix, i'll fix it in the other PR)